other/materials_designer/specific_examples/defect_surface_step_platinum.ipynb

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
<p>Let's move comments to lines above the vars</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>